coord/src/dist_aggregate.c

```diff
@@ -401,12 +401,6 @@ void RSExecDistAggregate(RedisModuleCtx *ctx, RedisModuleString **argv, int argc
   int rc = AREQ_Compile(r, argv + 2 + profileArgs, argc - 2 - profileArgs, &status);
   if (rc != REDISMODULE_OK) goto err;
 
-  // Save time when query was initiated
-  if (!r->reqTimeout) {
-    r->reqTimeout = RSGlobalConfig.queryTimeoutMS;
-  }
-  updateTimeout(&r->timeoutTime, r->reqTimeout);
-
```

Comment:
is this need to be removed?

Reply:
This removal was part of Ariel's PR, right? @ashtul

Reply:
@DvirDukhan @alonre24 It was added to function `AREQ_Compile`.